lib/pharos/ssh/remote_command.rb

```diff
@@ -122,29 +125,39 @@ def initialize_debug
         end
       end
 
+      # @return [Boolean]
       def debug?
         @debug
       end
 
+      # @param cmd [String]
+      # @param source [String, NilClass]
+      # @return [Integer]
       def debug_cmd(cmd, source: nil)
-        $stdout.write(INDENT + pastel.cyan("$ #{cmd}" + (source ? " < #{source}" : "")) + "\n")
+        $stdout.write("#{INDENT} #{pastel.cyan("#{@client.host}:")} #{pastel.cyan("$ #{cmd}" + (source ? " < #{source}" : ""))}\n")
```

Comment:
```suggestion
         $stdout.write("#{INDENT} #{pastel.cyan("#{@client.host}:")} #{pastel.cyan("$ #{cmd}" + (" < #{source}" if source))}\n")
```

should silence rubocop and skips empty string creation